loom/graphics/gfxTexture.cpp

```diff
@@ -277,6 +282,8 @@ void bitmapExtrudeRGBA_c(const void *srcMip, void *mip, int srcHeight, int srcWi
 
 TextureInfo *Texture::load(uint8_t *data, uint16_t width, uint16_t height, TextureID id)
 {
+	id &= TEXTURE_ID_MASK;
```

Comment:
Hmm, maybe. But at some places the id is generated through other means e.g. as the index from a for loop. We could enforce everything to use the above function, but I think that would mean that we also have to change the places where the id is generated. Thoughts?